### PR TITLE
Require a system test before automatically triggering a deploy

### PR DESCRIPTION
**Pull request by Rheeseyb**
**Problem:**
The automated deploy runs every morning regardless of the state of the staging environment, unless we explicitly deactivate it.

**Fix:**
Separate out the automated triggering of the Tag and Deploy workflow so that it can run a system test against the Staging environment before actually tagging and deploying the project.

### CODE REVIEW

File: .github/workflows/tag-release.yml [hunk 1]
```diff
@@ -8,12 +8,12 @@ on:
       commit-ref:
         description: 'Branch or SHA to tag (defaults to master)'
         required: false
-  schedule:
-    - cron: '0 5 * * 1-5' # Run every weekday 5:00 UTC
+  workflow_call:
```
Comment (ruggi) [inline .github/workflows/tag-release.yml:11]: I may be wrong, but I'm not sure this is needed 🤔

**Review by ruggi — APPROVED** [review]

**Review by seanparsons — APPROVED** [review]

File: .github/workflows/tag-release.yml (hunk 1 above)
Comment (Rheeseyb) [reply .github/workflows/tag-release.yml:11]: You are wrong 😀 https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_call
The `workflow_dispatch` event is for manually triggering the workflow

Comment (ruggi) [reply .github/workflows/tag-release.yml:11]: aha! TIL

Comment (Rheeseyb) [reply .github/workflows/tag-release.yml:11]: Me too!

**Pull request merged**